libs/blocks/hero-marquee/hero-marquee.js

```diff
@@ -0,0 +1,208 @@
+import {
+  // eslint-disable-next-line max-len
+  decorateBlockBg, decorateBlockHrs, decorateBlockText, decorateTextOverrides, decorateButtons, handleObjectFit,
+} from '../../utils/decorate.js';
+import { createTag, loadStyle, getConfig } from '../../utils/utils.js';
+
+const contentTypes = ['list', 'qrcode', 'lockup', 'text', 'bgcolor', 'supplemental'];
+const rowTypeKeyword = 'con-block-row-';
+const breakpointThemeClasses = ['dark-mobile', 'light-mobile', 'dark-tablet', 'light-tablet', 'dark-desktop', 'light-desktop'];
+
+function distillClasses(el, classes) {
+  const taps = ['-heading', '-body', '-detail'];
+  classes.forEach((elClass) => {
+    const elTaps = taps.filter((tap) => elClass.endsWith(tap));
+    if (!elTaps.length) return;
+    const parts = elClass.split('-');
+    el.classList.add(`${parts[1]}-${parts[0]}`);
+    el.classList.remove(elClass);
+  });
+}
+
+function decorateList(el, classes) {
+  el.classList.add('body-l');
+  const listItems = el.querySelectorAll('ul li', 'ol li');
```

Comment:
Do we have `li` elements outside of `ol` or `ul`? Could we not just look for `li`?

Reply:
No other `li` elements should be considered, a a simple `li` works fine. Updated